Makefile

```diff
@@ -4,7 +4,6 @@ GO_FILES := $(shell \
 	find . '(' -path '*/.*' -o -path './vendor' ')' -prune \
 	-o -name '*.go' -print | cut -b3-)
 
-GOLINT = $(GOBIN)/golint
```

Comment:
Maybe we can use [`revive`](https://github.com/mgechev/revive) instead of the deprecated `golint`?

Revive is the drop-in replacement for golint.